src/components/Navigation.js

```diff
@@ -1,19 +1,43 @@
 import React from 'react';
+import styled from "styled-components";
 
 const Navigation = () => {
   return (
-    <ul className="navigation">
+    <StyledNavigation >
       <li>
-        <a href="#">Homepage</a>
+        <StyledLink href="#">Homepage</StyledLink>
       </li>
       <li>
-        <a href="#">Favorites</a>
+        <StyledLink href="#">Favorites</StyledLink>
       </li>
       <li>
-        <a href="#">Ramdom</a>
+        <StyledLink href="#">Random</StyledLink>
       </li>
-    </ul>
+    </StyledNavigation>
   );
 };
 
+
+const StyledNavigation = styled.div`
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  list-style: none;
+  background-color: green;
+  height: 70px;
+  position: fixed;
+  bottom: 0px;
```

Comment:
when you are using `0px` you can just write the `0` and omit the `px` ;-)

Reply:
thank you for your detailed review. I've done all the changes.